src/test/java/com/spotify/docker/client/DockerHostTest.java

```diff
@@ -98,7 +98,7 @@ public void testCertPathFromEnv() throws Exception {
     DockerHost.setSystemDelegate(systemDelegate);
 
     assertThat(DockerHost.certPathFromEnv(), equalTo("foo"));
-    assertThat(DockerHost.certPathFromEnv(), equalTo("bar/.docker"));
+    assertThat(DockerHost.certPathFromEnv(), equalTo(null));
```

Comment:
Gotcha, that makes sense. In general I end up writing two separate tests for that.